lib/config/config-file.js

```diff
@@ -524,7 +527,7 @@ function resolve(filePath, relativeTo) {
 function loadFromDisk(resolvedPath, configContext) {
     const dirname = path.dirname(resolvedPath.filePath),
         lookupPath = getLookupPath(dirname);
-    let config = loadConfigFile(resolvedPath);
+    let config = loadConfigFile(resolvedPath, configContext);
```

Comment:
Oops, this was unintentional, thanks. While working on this I initially considered adding a second argument to `loadConfigFile`, but then I forgot to remove it here.